content/docs/command-reference/exp/run.md

```diff
@@ -84,13 +84,23 @@ steps as `make_checkpoint()` — please refer to its reference for details.
 You can now use `dvc exp run` to begin the experiment. If the process gets
 interrupted (e.g. with `[Ctrl] C` or by an error), all the checkpoints so far
 will be preserved. When a run finishes normally, a final checkpoint will be
-added (if needed) to wrap up the experiment.
+added (if needed) to wrap up the experiment. Following uses of `dvc exp run`
+will continue from this point (using the latest cached versions of all outputs).
 
-Following uses of `dvc exp run` will continue from this point (using the latest
-cached versions of all outputs). You can add a `--rev` to continue from a
-previous checkpoint instead (list them with `dvc exp show`). Or use `--reset` to
-start over (discards previous checkpoints and deletes `checkpoint` outputs, like
-the first `dvc exp run`) — useful for re-training ML models, for example.
+To continue from a previous checkpoint (list them with `dvc exp show`), when
+running experiments in your workspace, you must first use `dvc exp apply` to
+apply the checkpoint from which you want to resume before using `dvc exp run`.
+When running experiments outside of your workspace via `--queue/--run-all` or
+`--temp`, you can use `--rev` to directly specify the checkpoint from which you
+want to resume.
```

Comment:
```suggestion
List previous checkpoints with `dvc exp show`. To continue from a previous
checkpoint when running experiments in your workspace, you must first use
`dvc exp apply` to apply the checkpoint from which you want to resume before
using `dvc exp run`. When running experiments outside of your workspace via
`--queue/--run-all` or `--temp`, you can use `--rev` to directly specify the
checkpoint from which you want to resume.
```

Reply:
Improved for sure. But this section is about checkpoints, not about `exp show`. Shouldn't even be in `exp run` probably (WIP in #2373). Maybe the `exp show` brief mention should go earlier in some other paragraph?

Reply:
Let's remove ~~when running experiments in your workspace~~ ? The queue/temp section comes after this one so I doubt people will understand what it means. Maybe a paragraph like this instead "(except in `--queue/temp` runs, see next section)"

Reply:
"use `exp apply` to apply" sounds redundant I think.

Reply:
How about just "you must `exp apply` it first" ?

Reply:
Cc @casperdcl those ☝️ were meant as ideas for you to update your suggestion 🙂

Reply:
Also:
- ~~directly~~
- "from which you want to ~~resume~~ continue" (consistency)

Reply:
```suggestion
List previous checkpoints with `dvc exp show`. To continue from a previous
checkpoint, you must first `dvc exp apply` it before using `dvc exp run`. For
convenience, when running experiments outside of your workspace via
`--queue/--run-all` or `--temp`, you can use `--rev` instead to specify the
checkpoint from which you want to continue.
```

This is a command reference so I think it's fine to talk about `--queue/--run-all/--temp` without adding "(see below)".

Reply:
Nice, committing. BTW I'm just trying to give some recs these first times, but you're the reviewer  @casperdcl, so up to you 🙂 (unless someone detects errors or has strong opinions, in which case lets discuss as needed). Anyway, committing this one for now.

Reply:
Nice, committing. BTW I'm just trying to give some recs these first times, but you're the reviewer  @casperdcl, so up to you 🙂 (unless someone detects errors or has strong opinions, in which case lets discuss as needed).